### DISCUSSION TITLE
Default value for dates

### DISCUSSION THREAD

**Discussion by HuifangYeo** [atoti/atoti#671] [Q&A] [answered]
Hi,

with v0.7.0, dates are assigned default values of `1970-01-01`:
![image](https://user-images.githubusercontent.com/35210495/185534539-6e95fc36-cdfb-42b2-9b99-fe488fc08f90.png)

In my use case, having a date doesn't make sense but I don't want to filter member away either. It should be as follows as I still want to see the NCD:
![image](https://user-images.githubusercontent.com/35210495/185534836-f7545ee3-b068-48f4-9c19-34a2d082c49c.png)

Can we still set the date to N/A if it's not applicable?

**Comment by Skyscimitar** [comment, marked as answer]
Hi Huifang,

As of version 0.7.0 you can set the `default_value` attribute on the columns of your table when creating it, or directly on the column. See https://docs.atoti.io/latest/lib/atoti/atoti.column.html#atoti.Column.default_value and https://docs.atoti.io/latest/lib/atoti/atoti.column.html#atoti.Column.default_value for more information. Note that the `default_value` parameter has also been added to all the `read_*` methods.

So in your case when creating your table you could add `session.read_*(..., default_values={"CLAIM_DATE": "N/A"})`
